pom.xml

```diff
@@ -259,7 +259,9 @@
                                 <clearDefaultOptions>true</clearDefaultOptions>
                                 <options>
                                     <option>-arch</option>
-                                    <option>${nar.arch}</option>
+                                    <option>x86_64</option>
+                                    <option>-arch</option>
+                                    <opiton>arm64</option>
```

Comment:
```suggestion
                                    <option>arm64</option>
```

Reply:
I fixed this on behalf of the submitter.